backend/src/lib/jsonschema/api/RemoveUsers.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$comment": "Request object for the RemoveUsers endpoint",
+  "type": "object",
+  "properties": {
+    "usernames": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    }
+  },
+  "required": ["usernames"]
+}
```

Comment:
@trietlu @miguelpdiaz8 We would need to define the json schema for the Request object for each of our endpoints like this. The nice thing is that we should be able to leverage these schemas when we build our Open API documentation since it supports `jsonschema`.